src/Model/Operation/AssetFundPoolsOperation.php

```diff
@@ -111,8 +112,10 @@ public function toBytes(): string
             $this->getFrom()->toBytes(),
             $this->getUia()->toBytes(),
             $this->getDct()->toBytes(),
-            // TODO: Extensions Array
-            $this->getExtensions() ? '01' : '00'
+            $this->getExtensions() ?
+                VarInt::encodeDecToHex(sizeof($this->getExtensions()))
+                . '' // TODO array_map each element toBytes()
```

Comment:
unresolved todo

Reply:
Unknown structure of array, can't fix now.